### ISSUE TITLE
preprocessors: autoprefixer: included in examples before it's installed; its example doesn't include postcss-import

### ISSUE THREAD

**Issue by olets** [tailwindlabs/tailwindcss.com#1894]
Couple misses on the ["Using with Preprocessors" page](https://tailwindcss.com/docs/using-with-preprocessors):

1. autoprefixer is included in the sample postcss configs (e.g. https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports) before the example that installs it (https://tailwindcss.com/docs/using-with-preprocessors#vendor-prefixes)
2. The "Vendor prefixes" example (again, https://tailwindcss.com/docs/using-with-preprocessors#vendor-prefixes) doesn't properly build on the minimum requirements in the same way other examples do: it does not include `postcss-import`.

**Comment by adamwathan** [comment]
Hey! Those examples are not necessarily meant to be a "step 1", "step 2", sort of thing — they are just docs for different independent concerns.

Including autoprefixer is part of all the installation documentation (https://tailwindcss.com/docs/installation/using-postcss) so I don't feel like it's surprising that it's included there personally, and I think it's helpful to have more than one other plugin there to help really communicate that `postcss-import` has to be _first_ in the list.

The vendor prefixes section doesn't require `postcss-import` to be installed so it's not included in the list — you can build a site without `postcss-import` if you want.

Thanks for the feedback regardless — if this is something that trips up a lot of people we can rethink it.

**Comment by olets** [comment]
Aha. It's a long time since I used PostCSS with Tailwind, and forgot that there's a dedicated, different page. Searching via search engine I landed on the Using with preprocessors, and didn't wonder whether there was other PostCSS documentation not surfaced in the sidebar. Looking at the two together, I'd expect Using with Preprocessors to

- start with "Set up PostCSS per this other page…"
- not have the duplicative vendor prefixes section
- include autoprefixer in its postcss-preset-env example
